backend/core/types/src/backend-swagger.ts

```diff
@@ -700,6 +700,21 @@ export class AssetsService {
       axios(configs, resolve, reject)
     })
   }
+  /**
+   * Upload asset
+   */
+  upload(options: IRequestOptions = {}): Promise<Asset> {
```

Comment:
is this related to the bathrooms filtering?

Reply:
It came up when I regenerated the client, which I guess was using a different version of the backend.  I suspect it was for PR #96 which just got merged, so it should accurately reflect the backend state.